src/VisualStudio/Core/Def/Implementation/ProjectSystem/InvisibleEditor.cs

```diff
@@ -28,6 +28,12 @@ internal partial class InvisibleEditor : IInvisibleEditor
         private OLE.Interop.IOleUndoManager _manager;
         private readonly bool _needsUndoRestored;
 
+        [Obsolete("This is a compatibility shim for Live Share; please do not use it.")]
```

Comment:
📝 IVTs to external assemblies are use-at-own-risk. Consumers are expected to update code to address changes we make, if and when such changes are made.